application/classes/Ushahidi/Repository/Post/Export.php

```diff
@@ -30,7 +30,7 @@ public function retrieveColumnNameData($data) {
 	foreach ($data['values'] as $key => $val)
     {
         $attribute = $this->form_attribute_repo->getByKey($key);
-        $attributes[$key] = ['label' => $attribute->label, 'priority'=> $attribute->priority, 'stage' => $attribute->form_stage_id, 'type'=> $attribute->type, 'form_id'=> $data['form_id']];
+        $attributes[$key] = ['label' => $attribute->label, 'input' => $attribute->input, 'priority'=> $attribute->priority, 'stage' => $attribute->form_stage_id, 'type'=> $attribute->type, 'form_id'=> $data['form_id']];
```

Comment:
this is a bit weird.. but I couldn't find another way to know if it's actually a date or datetime other than using 'input' . The type only says datetime